src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -623,7 +623,7 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         /// <param name="imputeMissingLabelsAsNegative">Whether to treat missing labels as having negative labels, instead of keeping them missing.</param>
         /// <param name="maxCalibrationExamples">Number of instances to train the calibrator.</param>
         /// <typeparam name="TModel">The type of the model. This type parameter will usually be inferred automatically from <paramref name="binaryEstimator"/>.</typeparam>
-        public static Pkpd PairwiseCoupling<TModel>(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
+        public static PkpdTrainer PairwiseCoupling<TModel>(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
```

Comment:
>PkpdTrainer PairwiseCoupling [](start = 22, length = 28)

`PairwiseCouplingTrainer PairwiseCoupling` Let's match name of trainer and method in catalog

Reply:
Ok thanks, I renamed pkpd to PairwiseCouplingTrainer

---
In reply to: [260530151](https://github.com/dotnet/machinelearning/pull/2749#discussion_r260530151) [](ancestors = 260530151)